docs/output-files/log-simulation-parameters.md

```diff
@@ -1,25 +1,201 @@
 ---
 layout: default
```

Comment:
Ok, so there is the same file extension for sim and processing logs, right? Should we change this?

Reply:
No need to have one extension per file, especially here, both file play the same role but for a different module. Having files with many different extensions can be discouraging